SPECS/fwctl/fwctl.spec

```diff
@@ -250,6 +250,9 @@ fi # 1 : closed
 %endif
 
 %changelog
+* Wed Feb 19 2025 Chris Co <chrco@microsoft.com>
```

Comment:
Missing version & release. Need to find out why rpmdev-bumpspec didn't automatically add it...